data/other/spectre-meltdown-checker.sh

```diff
@@ -0,0 +1,980 @@
+#! /bin/sh
```

Comment:
I would prefer a `git subtree` as this script is going to evolve and is not going to become a Debian package as for `acme-tiny`.